src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3329,7 +3330,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         ExecuteAsTool="$(ResGenExecuteAsTool)"
         EnvironmentVariables="$(ResGenEnvironment)"
         MSBuildRuntime="$(GenerateResourceMSBuildRuntime)"
-        MSBuildArchitecture="$(GenerateResourceMSBuildArchitecture)">
+        MSBuildArchitecture="$(GenerateResourceMSBuildArchitecture)"
+        WarnOnBinaryFormatterUse="$(GenerateResourceWarnOnBinaryFormatterUse)"
```

Comment:
By convention, put this real input above the virtual inputs `MSBuildRuntime` and `MSBuildArchitecture`.